wiki/People/Global_Moderation_Team/en.md

```diff
@@ -16,38 +16,41 @@ The Global Moderation Team is responsible for the welfare of the chat/forum and
 
 | Name | Language(s) | Area of focus |
 | :-- | :-- | :-- |
-| ![][flag_IT] [-kevincela-](https://osu.ppy.sh/users/266596) | Italian | Chat Moderation |
 | ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | | Chat Moderation |
 | ![][flag_DE] [Clobohne](https://osu.ppy.sh/users/499343) | German | Chat Moderation |
 | ![][flag_AR] [Darksonic](https://osu.ppy.sh/users/570042) | Spanish | Chat Moderation, Forum Moderation |
 | ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) | | Chat Moderation, Technical Support |
 | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565) | Spanish | Tournaments |
 | ![][flag_US] [Dntm8kmeeatu](https://osu.ppy.sh/users/5428812) | | Technical Support |
 | ![][flag_BR] [Edu](https://osu.ppy.sh/users/5618109) | Portuguese, Japanese | Chat Moderation |
-| ![][flag_KR] [Elfin](https://osu.ppy.sh/users/1399551) | Korean, Japanese | Chat Moderation, Forum Moderation |
 | ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | | Player Support |
 | ![][flag_BG] [Flanster](https://osu.ppy.sh/users/447818) | Bulgarian | osu!idol hosting, Forum Moderation |
 | ![][flag_CN] [Fycho](https://osu.ppy.sh/users/1876867) | Chinese | Chat Moderation, Forum Moderation |
 | ![][flag_CA] [Gabe](https://osu.ppy.sh/users/654108) | French | Chat Moderation, Forum Moderation |
 | ![][flag_PL] [Galkan](https://osu.ppy.sh/users/169570) | Polish | Chat Moderation |
 | ![][flag_BR] [ghm12](https://osu.ppy.sh/users/2594229) | Portuguese | Chat Moderation |
 | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779) | Turkish | Chat Moderation |
+| ![][flag_ID] [Hinsvar](https://osu.ppy.sh/users/1249323) | Indonesian | Chat Moderation, BSS Moderation |
```

Comment:
probably `Beatmap Moderation` instead

Reply:
This is due to to the fact that we will moderate elements of the beatmaps themselves, just the content of the uploads.

Reply:
not sure what the difference is but either way you're definitely not moderating BSS itself so... :thinking:

Reply:
"Beatmap report processing" or something?

Reply:
I will go with that